[1.265s] Traceback (most recent call last):
  File "C:\Users\yeira\miniforge3\envs\humble\Lib\site-packages\colcon_core\executor\__init__.py", line 91, in __call__
    rc = await self.task(*args, **kwargs)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\yeira\miniforge3\envs\humble\Lib\site-packages\colcon_core\task\__init__.py", line 93, in __call__
    return await task_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\yeira\miniforge3\envs\humble\Lib\site-packages\colcon_ros\task\ament_python\build.py", line 102, in build
    return await extension.build(additional_hooks=additional_hooks)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\yeira\miniforge3\envs\humble\Lib\site-packages\colcon_core\task\python\build.py", line 143, in build
    temp_symlinks = self._symlinks_in_build(args, setup_py_data)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\yeira\miniforge3\envs\humble\Lib\site-packages\colcon_core\task\python\build.py", line 358, in _symlinks_in_build
    os.symlink(src, dst)
OSError: [WinError 1314] El cliente no dispone de un privilegio requerido: 'C:\\dev\\p-p_teleop_robotics_unal\\src\\joy_controller\\setup.py' -> 'C:\\dev\\p-p_teleop_robotics_unal\\build\\joy_controller\\setup.py'
